### DISCUSSION TITLE
Ruleset leaderboards not supported message instead of unable to fetch scores.

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#17041] [Ideas] [answered]
Just a quick idea proposal to reduce some confusion. I propose instead of displaying, "Unable to Fetch Scores" message on leaderboards in lazer it should display, "This Ruleset doesn't support leaderboards currently, Check back later" or similar.

The main reason I propose this minor change is to reduce confusion when osu opens up with a custom ruleset selected instead of an official ruleset. I was about to make a different discussion for scores not fetching before I realized I was on Yoso mode and not osu!. This is a low priority issue but I believe it will reduce some of the confusion

**Comment by frenzibyte** [comment, marked as answer]
Should be resolved by https://github.com/ppy/osu/pull/17042.
